packages/site/cms-site/package.json

```diff
@@ -1,21 +1,28 @@
 {
     "name": "@comet/cms-site",
+    "type": "module",
```

Comment:
We should verify that changing this can be treated as a non-breaking change. It probably will work in our projects.

Reply:
Doesn't matter because I'll only keep this for the split packages, not for the current cms-site Package. Same for the CSS file in the next PR.

Now, it goes to the feature branch. Before merging the feature branch, I'll copy the old cms-site back in so it remains unchanged.

(Also it will work in our projects)